Bigtable/src/Connection/Grpc.php

```diff
@@ -325,15 +337,67 @@ public function getTable(array $args)
      */
     public function deleteTable(array $args)
     {
-        throw new \BadMethodCallException('This method is not implemented yet');
+        $name = $this->pluck('name', $args);
+        return $this->send([$this->bigtableTableAdminClient, 'deleteTable'], [
+            $name,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
     }
 
     /**
      * @param array $args
      */
     public function modifyColumnFamilies(array $args)
     {
-        throw new \BadMethodCallException('This method is not implemented yet');
+        $name = $this->pluck('name', $args);
+        $modifications = $this->pluck('modifications', $args);
+        return $this->send([$this->bigtableTableAdminClient, 'modifyColumnFamilies'], [
+            $name,
+            array_map([$this,'columnFamilyModificationsObject'], $modifications),
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $modifications
+     * @return Modification
+     */
+    private function columnFamilyModificationsObject(array $modifications)
+    {
+
+        if (isset($modifications['create'])) {
+            $create = $this->pluck('create', $modifications);
+            $modifications['create'] = $this->columnFamilyObject($create);
```

Comment:
We don't need to use `pluck` here, since we're simply re-assigning on the next line. Try this in place of the pluck calls in this method:

```php
$modifications['create'] = $this->columnFamilyObject($modifications['create']);
```

Reply:
When I do that, empty array is getting converted to null while calling method columnFamilyObject and it fails with below error.

TypeError: Argument 1 passed to Google\Cloud\Bigtable\Connection\Grpc::columnFamilyObject() must be of the type array, null given, called in /var/www/google-cloud-php/Bigtable/src/Connection/Grpc.php on line 370